WindowsDevicePortalWrapper/WindowsDevicePortalWrapper/DefaultDevicePortalConnection.cs

```diff
@@ -45,6 +44,29 @@ public class DefaultDevicePortalConnection : IDevicePortalConnection
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="XboxDevicePortalConnection"/> class, using a SecureString to secure the password.
```

Comment:
OIC, instead of "XboxDevicePortalConnection" it should be "DefaultDevicePortalConnection"